modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/describer/model/ConfigurationElement.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.extension.internal.introspection.describer.model;
+
+/**
+ * A contract for an element to be considered as an extension configuration
```

Comment:
to be considered as an extension configuration -> from which an extension configuration can be derived